Frontend/CO.CDP.OrganisationApp/Session.cs

```diff
@@ -6,32 +6,33 @@ public class Session(IHttpContextAccessor httpContextAccessor) : ISession
     {
         public T? Get<T>(string key)
         {
-            CheckSessionIsNull();
-
-            var value = httpContextAccessor.HttpContext!.Session.GetString(key);
+            var value = GetSession().GetString(key);
```

Comment:
This is a bonus change. 

Asking for a validated value is less error-prone and should be easier to read than a validation method followed by the access to validated value.

Previously, `CheckSessionIsNull` validated if `httpContextAccessor.HttpContext!.Session` can be accessed, and then the `Get` method accessed the value. These two lines of code are of different levels of abstraction.